src/Feed/EventPublisher.ts

```diff
@@ -54,14 +54,25 @@ export default function useEventPublisher() {
                 return match
             }
         }
+        const replaceNoteId = (match: string) => {
+            try {
+                const hex = bech32ToHex(match);
+                const idx = ev.Tags.length;
+                ev.Tags.push(new Tag(["e", hex, "mention"], idx));
```

Comment:
The mention marker comes after the relay hint it should be `["e", hex, "", "mention"]`